modules/cnf-gathering-data-about-cluster-using-must-gather.adoc

```diff
@@ -32,9 +32,9 @@ $ oc adm must-gather --image=<PAO_must_gather_image> --dest-dir=<dir>
 +
 .Example
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather --image=registry.redhat.io/openshift4/performance-addon-operator-must-gather-rhel8:v4.11 --dest-dir=must-gather
+$ oc adm must-gather --image=registry.redhat.io/openshift4/performance-addon-operator-must-gather-rhel8:v{product-version} --dest-dir=must-gather
```

Comment:
dest-dir=<path_to-must-gather>
here too?

Reply:
Hi @yuvalk thanks for feedback I picked this up. Hopefully good for LGTM